lms/templates/learner_dashboard/expired_notification.underscore

```diff
@@ -0,0 +1,2 @@
+<span><i class="fa fa-info-circle fa-lg" aria-hidden="true"></i></span>
+<span>You enrolled in this course previously but did not earn the certificate required to complete this program.</span>
```

Comment:
Strike "previously." It doesn't add anything, and might even confuse people who are enrolled in an open run.